lib/front_matter_parser/syntax_writer.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
```

Comment:
This file should be included from `front_matter_parser.rb`, so requiring last one makes all library available.